# Identify the team (when applicable)

uint8 FRIENDS = 0
uint8 FOES = 1
uint8 NONE = 2

uint8 data
